docs/VTX.md

```diff
@@ -138,6 +138,15 @@ vtxtable powerlabels 25 200 500 800
 #### SmartAudio V2.1 devices vary depending on their model. Check the manufacturers website.
 For these devices the `powervalues` are the output power in dBm.
 
+To query the available power levels from a SmartAudio 2.1 VTX enter the `vtxtable powerlevels` command with no parameters. This will report the available power settings suitable for pasting into the CLI thus:
```

Comment:
The code looks good now, but this is now inaccurate.

Reply:
Fixed